src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(':')
```

Comment:
The `split(':')` should be `split(';')` to correctly parse the key-value pairs. The task requires splitting by semicolons first to separate different key-value pairs.